internal/operator-controller/rukpak/convert/helm.go

```
@@ -17,12 +18,24 @@ type BundleToHelmChartConverter struct {
 	IsWebhookSupportEnabled bool
 }
 
-func (r *BundleToHelmChartConverter) ToHelmChart(bundle source.BundleSource, installNamespace string, watchNamespace string) (*chart.Chart, error) {
+func (r *BundleToHelmChartConverter) ToHelmChart(bundle source.BundleSource, config map[string]interface{}) (*chart.Chart, error) {
 	rv1, err := bundle.GetBundle()
 	if err != nil {
 		return nil, err
 	}
 
+	// Convert config map to render options
+	var opts []render.Option
+	if config != nil {
+		if installNs, ok := config[bundlepkg.BundleConfigInstallNamespaceKey].(string); ok {
```

we should return an error here as installNamespace is mandatory.